### ISSUE TITLE
Bug on fresh installation

### ISSUE THREAD

**Issue by johnnyschneider** [msys2/MSYS2-packages#2295]
## Describe the issue

I believe this is a bug in the fresh installation process. But it could be intentional - I'm not sure.

A clear and concise description of what the issue is.

![image](https://user-images.githubusercontent.com/22988624/103434413-86140980-4bc6-11eb-94f3-06dd482c2248.png)

To be clear, it does _not_ affect the end result. msys was still fully installed.

## Steps to Reproduce the Problem

1. Run the installer exactly as the website says to.
3. This text appears

(1/1) Updating the info directory file...
      0 [main] pacman (5468) C:\msys64\usr\bin\pacman.exe: *** fatal error - cygheap base mismatch detected - 0x18034B408/0x180345408.
This problem is probably due to using incompatible versions of the cygwin DLL.
Search for cygwin1.dll using the Windows Start->Find/Search facility
and delete all but the most recent version.  The most recent version *should*
reside in x:\cygwin\bin, where 'x' is the drive on which you have
installed the cygwin distribution.  Rebooting is also suggested if you
are unable to find another cygwin DLL.
      0 [main] pacman 2203 dofork: child -1 - forked process 5468 died unexpectedly, retry 0, exit code 0xC0000142, errno 11
error: could not fork a new process (Resource temporarily unavailable)
:: To complete this update all MSYS2 processes including this terminal will be closed. Confirm to proceed [Y/n]

4. What did you expect to happen instead? In the past, I used to get just the last message. The majority of the paragraph is new to me. I usually just see the sentence "To complete this update all MSYS2 processes including this terminal will be closed. Confirm to proceed [Y/n]"

## Additional Context: Operating System, Screenshots

- OS: Windows 10 Pro 20H2 64bit

**Comment by Biswa96** [comment]
It is not really a bug but feature of Microsoft Windows OS. As pacman is updating bash and mintty packages it has to replace those binaries which are running at that time. In Windows you can not replace a already running binary _in straight way_. So, the result is garbage output.

**Comment by jeremyd2019** [comment]
I saw this too.  I'm guessing one of the core packages (bash?) installs info pages, that cause the pacman trigger to run, but bad things happen if you `fork` after msys2-runtime is updated (hence the prompt to terminate all running processes)

**Comment by johnnyschneider** [comment]
Cool beans. Happy new years!

**Comment by lazka** [comment]
I guess because the hook calls into bash again? Maybe we can use dash instead?

**Comment by jeremyd2019** [comment]
I don't think that matters - I think any fork after msys-2.0.dll has been updated will do that.

**Comment by lazka** [comment]
hm, I see, tricky..
